test/ibmq/test_ibmq_job.py

```diff
@@ -412,12 +412,12 @@ def test_retrieve_jobs_db_filter(self):
         qc = QuantumCircuit(3, 3)
         qc.h(0)
         qc.measure([0, 1, 2], [0, 1, 2])
-        job = self._sim_job(qc=qc)
-        job.wait_for_final_state()
+        job = self._run_job(qc=qc)
+        cancel_job(job)
 
         my_filter = {'backend.name': self.sim_backend.name(),
                      'summaryData.summary.qobj_config.n_qubits': 3,
-                     'status': 'COMPLETED'}
+                     'status': 'CANCELED'}
```

Comment:
Same as above. Using `CANCELLED` (the API status has 2 L's, btw) can be flaky.

Reply:
[Restored to original!](https://github.com/Qiskit-Partners/qiskit-ibm/pull/9/files#diff-313b7cac7866bc872344a595d0f613f12c5dd1bd6ade62031c8f3e5f7e4330d0R418-R423)